File: app/src/main/java/com/meteoro/testeandroid/core/data/model/Screen.java
package com.meteoro.testeandroid.core.data.model;

import java.util.List;

import lombok.Getter;
import lombok.experimental.Accessors;

@Accessors(fluent = true, chain = true)
public class Screen {

    @Getter
    String title;

    @Getter
    String fundName;

    @Getter
    String whatIs;

    @Getter
    String definition;

    @Getter
    String riskTitle;

    @Getter
    int risk;
    String infoTitle;

    @Getter
    MoreInfo moreInfo;

    @Getter
    List<Info> info;

    @Getter
    List<DownInfo> downInfo;
}

File: InvestApp/app/src/main/java/com/study/vipoliveira/investapp/data/network/investment/entities/Screen.kt
package com.study.vipoliveira.investapp.data.network.investment.entities

data class Screen (
        val title: String,
        val fundName: String,
        val whatIs: String,
        val definition: String,
        val riskTitle: String,
        val risk: Int,
        val infoTitle: String,
        val moreInfo: MoreInfo,
        val info: List<Info>,
        val downInfo: List<Info>
)